client/blocks/video-editor/video-editor-buttons.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import UploadButton from './video-editor-upload-button';
+
+const VideoEditorButtons = ( {
+	isPosterUpdating,
+	isVideoLoading,
+	onCancel,
+	onSelectFrame,
+	onUploadImage,
+	onUploadImageClick,
+	translate,
+} ) => {
+	return (
+		<div className="video-editor__buttons">
+			{ onCancel &&
+				<Button
+					className="video-editor__buttons-button"
```

Comment:
Perhaps rename the component VideoEditorControls, so this can have a less buttony name 😄 eg something like `video-editor__controls-button`